### PR TITLE
* Resolved #226 & #268

### LINKED ISSUE #226: [Bug]: OutlookGridView throws a MissingManifestResourceException when trying to configure its columns
**Describe the bug**
I've been using an old version of `OutlookGridView` and just tried to upgrade to the [latest NuGet package](https://www.nuget.org/packages/Krypton.Toolkit.Suite.Extended.Outlook.Grid/5.550.2106). When I ran the application, it threw a `MissingManifestResourceException`.

**To Reproduce**
1. Add an `OutlookGridView` control to the form and add a couple of columns to it.
2. In the form's constructor add the following code:
```
foreach (DataGridViewColumn col in kryptonOutlookGrid1.Columns)
{
    col.SortMode = DataGridViewColumnSortMode.Programmatic;
    kryptonOutlookGrid1.AddInternalColumn(
        col, new OutlookGridDefaultGroup(null), SortOrder.None, -1, -1);
}
```
3. Run the program.
4. See error.

Here's the full stack trace:

```
System.Resources.MissingManifestResourceException
  HResult=0x80131532
  Message=Could not find any resources appropriate for the specified culture or the neutral culture.  Make sure "Krypton.Toolkit.Suite.Extended.Outlook.Grid.Utilities.Language.Strings.resources" was correctly embedded or linked into assembly "Krypton.Toolkit.Suite.Extended.Outlook.Grid" at compile time, or that all the satellite assemblies required are loadable and fully signed.
  Source=mscorlib
  StackTrace:
   at System.Resources.ManifestBasedResourceGroveler.HandleResourceStreamMissing(String fileName)
   at System.Resources.ManifestBasedResourceGroveler.GrovelForResourceSet(CultureInfo culture, Dictionary`2 localResourceSets, Boolean tryParents, Boolean createIfNotExists, StackCrawlMark& stackMark)
   at System.Resources.ResourceManager.InternalGetResourceSet(CultureInfo requestedCulture, Boolean createIfNotExists, Boolean tryParents, StackCrawlMark& stackMark)
   at System.Resources.ResourceManager.InternalGetResourceSet(CultureInfo culture, Boolean createIfNotExists, Boolean tryParents)
   at System.Resources.ResourceManager.GetString(String name, CultureInfo culture)
   at Krypton.Toolkit.Suite.Extended.Outlook.Grid.LanguageManager.GetLocalisedString(String name)
   at Krypton.Toolkit.Suite.Extended.Outlook.Grid.OutlookGridDefaultGroup..ctor()
   at Krypton.Toolkit.Suite.Extended.Outlook.Grid.OutlookGridDefaultGroup..ctor(IOutlookGridGroup parentGroup)
   at TmpOutlookGridNewTest.Form1..ctor() in \TmpOutlookGridNewTest\TmpOutlookGridNewTest\Form1.cs:line 23
   at TmpOutlookGridNewTest.Program.Main() in \TmpOutlookGridNewTest\TmpOutlookGridNewTest\Program.cs:line 19
```

**Environment:**
 - OS: Windows 10 build 19043.
 - .NET Framework 4.7.2.
 - OutlookGrid module version: 5.550.2106.

### PR DESCRIPTION
**Pull request by Wagnerp**
* Resolved #226 & #268
* Refactoring

### CODE REVIEW

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Custom Columns/TextAndImage.cs [hunk 1]
```diff
@@ -26,11 +26,11 @@ public class TextAndImage : IComparable<TextAndImage>
         /// <summary>
         /// The text
         /// </summary>
-        public string Text;
+        public string _text;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Custom Columns/TextAndImage.cs:29]: These are public variables, so should be in Capitals.

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Custom Columns/TokenListCell.cs [hunk 2]
```diff
@@ -88,7 +88,7 @@ protected override Size GetPreferredSize(Graphics graphics, DataGridViewCellStyl
             Size tmpSize = base.GetPreferredSize(graphics, cellStyle, rowIndex, constraintSize);
             Font f = KryptonManager.CurrentGlobalPalette.GetContentShortTextFont(PaletteContentStyle.GridDataCellList, PaletteState.Normal);
             int nextPosition = (int)(1 * factorX);
-            if (this.Value != null)
+            if (Value != null)
             {
                 foreach (Token tok in (List<Token>)this.Value)
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Custom Columns/TokenListCell.cs:93]: Missed a `this.` ;-)

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/Params/BarParams.cs [hunk 3]
```diff
@@ -27,15 +27,15 @@ public class BarParams : IFormatParams
         /// <summary>
         /// The bar color
         /// </summary>
-        public Color BarColour;
+        public Color _barColour;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/Params/BarParams.cs:30]: These are public.. Do not make them look like private fields !!

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/Params/ThreeColoursParams.cs [hunk 4]
```diff
@@ -29,22 +29,22 @@ public class ThreeColoursParams : IFormatParams
         /// <summary>
         /// The minimum color
         /// </summary>
-        public Color MinimumColour;
+        public Color _minimumColour;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/Params/ThreeColoursParams.cs:32]: Public not private.. I'll say no more, Just undo the changes please.

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/General/OutlookGridAlphabeticGroup.cs [hunk 5]
```diff
@@ -72,9 +66,13 @@ public override object Value
             {
                 //Note : value with Clone() is already 1 character, but no problem here
                 if (value != null && !string.IsNullOrEmpty(value.ToString())) //useful for textand image object
+                {
                     base.Value = value.ToString().Substring(0, 1).ToUpper();
+                }
                 else
+                {
                     base.Value = "";
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/General/OutlookGridAlphabeticGroup.cs:74]: Should state `string.Empty` so this does not look like some forgotten string.

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Custom Columns/TextAndImage.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Custom Columns/TextAndImage.cs]: Reverted the changes

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Custom Columns/TokenListCell.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Custom Columns/TokenListCell.cs:93]: Removed

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/Params/BarParams.cs (hunk 3 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/Params/BarParams.cs:30]: Reverted

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/Params/ThreeColoursParams.cs (hunk 4 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/Params/ThreeColoursParams.cs:32]: Reverted

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/General/OutlookGridAlphabeticGroup.cs (hunk 5 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/General/OutlookGridAlphabeticGroup.cs:74]: Changed

**Pull request merged**